package.json

```diff
@@ -19,10 +19,10 @@
     "cloudinary": "^1.27.1",
     "csso": "^5.0.2",
     "eleventy-plugin-toc": "^1.1.5",
+    "html-rewriter-wasm": "^0.4.1",
```

Comment:
Is this needed if it's loaded from a URL import?

Reply:
nope! That's from before I realized a URL import was available.